### PR TITLE
refactor: moved links from css files to html

### PR DESCRIPTION
**Pull request by nshanbhag20001412**
- Had imported CSS links into CSS files. This was creating a mess, so moved these links to respective HTML files.
- Moved color variables to text-utilities as of now

### CODE REVIEW

File: CSS/docs.css [hunk 1]
```diff
@@ -111,11 +105,33 @@
     scrollbar-width: 1px;
     width:100%; 
     height:100vh;
+    /* height:fit-content; */
+    /* height:max-content; */
+    /* overflow-y:none; */
+
+
```
Comment (pritam-kr) [inline CSS/docs.css]: remove this commented code

**Review by pritam-kr — APPROVED** [review]

File: CSS/main.css [hunk 2]
```diff
@@ -1,73 +1,28 @@
...
+    font-weight: 900;
+    text-align: center;
+}
+.hero-para{
+   margin-top: 2rem;
+   text-align: center;
+}
+.hero-buttons{
+  display:inherit;
+  margin:2rem;
```
Comment (Varu98) [inline CSS/main.css:26]: maybe this formatting code be improved ?

File: Component/Text-Utilies/text-utilities.css [hunk 3]
```diff
@@ -65,4 +101,11 @@ body {
     }
 } 
   */
+/* @media screen and (min-width : 1536px){
+
+    html{
+        font-size:150%;
+    }
+} 
+  */
```
Comment (Varu98) [inline Component/Text-Utilies/text-utilities.css:110]: this could be cleared

File: Component/button/button.css [hunk 4]
```diff
@@ -1,5 +1,3 @@
-@import url(/CSS/docs.css);
-
 
 .btn{
```
Comment (Varu98) [inline Component/button/button.css:2]: just found this class undeclared , this could be considered during refactoring later

**Review by Varu98 — CHANGES_REQUESTED** [review]
nothing major was found, all these issues can be resolved during a refactor later on 👍🏻

File: CSS/main.css (hunk 2 above)
Comment (nshanbhag20001412) [reply CSS/main.css:26]: can you give an example

File: Component/button/button.css (hunk 4 above)
Comment (nshanbhag20001412) [reply Component/button/button.css:2]: didnt get you

File: Component/Text-Utilies/text-utilities.css (hunk 3 above)
Comment (nshanbhag20001412) [reply Component/Text-Utilies/text-utilities.css:110]: i kept them for later stages of making website responsive

**Review by Varu98 — APPROVED** [review]
Awesome

**Review by tanveertkd — COMMENTED** [review]
Looks good to go!

**Pull request merged**